scripts/local-docker.env

```diff
@@ -3,14 +3,6 @@ export MARLOWE_CHAIN_SYNC_PORT=$(docker-compose port marlowe-chain-sync 3715 | s
 export MARLOWE_CHAIN_SYNC_QUERY_PORT=$(docker-compose port marlowe-chain-sync 3716 | sed -e s/://)
 export MARLOWE_CHAIN_SYNC_COMMAND_PORT=$(docker-compose port marlowe-chain-sync 3720 | sed -e s/://)
 
-export MARLOWE_RT_SYNC_HOST=127.0.0.1
```

Comment:
Do we really want to completely hide these? I'm wondering if there are some use cases where a client might want to use a service directly.

Reply:
The ports are still exposed, they would just need to be looked up manually (I don't want to encourage it)